#> cat pathes.msg

string path_name
bool empty
path[] pathes_data
